get_debug.py

```diff
@@ -2,44 +2,58 @@
 import os
 import tarfile
 import argparse
+import re
 from time import gmtime, strftime
-
 cwdir = os.getcwd()
 run_mode = "container"
 if(os.path.isfile(os.path.join(cwdir, 'ansible.cfg'))):
     run_mode = "repo"
-
 now = strftime("%Y-%m-%d-%H-%M-%S", gmtime())
 parser = argparse.ArgumentParser(description='This script collects the debug collateral in a tarball')
 parser.add_argument('--tarFileName', default='debug-' + now)
 parser.add_argument('--deploymentName')
 args = parser.parse_args()
 tarFileName = args.tarFileName
+print("The name of the tar file created:" + tarFileName + ".tar.gz")
 if args.deploymentName:
     deploymentName = os.path.join("deployments", args.deploymentName)
 else:
     deploymentName = "deployments"
-
 if run_mode == "repo":
     ansibleLogPath = os.path.join(cwdir, 'ansible.log')
     deploymentPath = os.path.join(cwdir, deploymentName)
     inventoryPath = os.path.join(cwdir, 'src/inventory')
+    print('The log path is:' + cwdir + '/metroae_data/' + tarFileName + ".tar.gz")
```

Comment:
If this is being run in the repo, the path would not contain "metroae_data".  That is a container thing.  Please also use os.path.join() for combining directories.